libpromises/eval_context.c

```diff
@@ -1751,6 +1765,15 @@ bool EvalContextClassPutSoftNS(EvalContext *ctx, const char *ns, const char *nam
     return EvalContextClassPut(ctx, ns, name, true, scope, tags);
 }
 
+/**
+ * Takes over #tags in case of success.
```

Comment:
```C
/**
 * Takes ownership of #tags in case of success.
 */
```

Reply:
I don't see the difference between the two sentences.

Reply:
The latter sounded more precise in my head. So, I just thought I'd suggest it :wink: